services/WAF/templates/waf-api-deploy.toml

```diff
@@ -0,0 +1,11 @@
+version = 0.1
+[default]
+[default.deploy]
+[default.deploy.parameters]
+stack_name = "waf-apigateway"
+s3_bucket = "aws-sam-cli-managed-default-samclisourcebucket-ei1wdt2g2rw4"
+s3_prefix = "waf-apigateway"
+region = "us-east-1"
+confirm_changeset = true
+capabilities = "CAPABILITY_IAM CAPABILITY_NAMED_IAM"
+parameter_overrides = "ActivateSqlInjectionProtectionParam=\"yes\" ActivateCrossSiteScriptingProtectionParam=\"yes\" ActivateHttpFloodProtectionParam=\"yes - AWS WAF rate based rule\" ActivateScannersProbesProtectionParam=\"yes - AWS Lambda log parser\" ActivateReputationListsProtectionParam=\"yes\" ActivateBadBotProtectionParam=\"yes\" EndpointType=\"ALB\" AppAccessLogBucket=\"api-accesslog-waf\" RequestThreshold=\"100\" ErrorThreshold=\"50\" WAFBlockPeriod=\"240\" KeepDataInOriginalS3Location=\"No\""
```

Comment:
You could just define the parameters in the template yaml file directly so you don't have to define the overrides here.